eng/Testing.props

```diff
@@ -4,6 +4,5 @@
     <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --blame-hang-dump-type full</TestRunnerAdditionalArguments>
     <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --blame-hang-timeout 10min</TestRunnerAdditionalArguments>
     <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --blame-crash-dump-type full</TestRunnerAdditionalArguments>
-    <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --blame-crash-collect-always</TestRunnerAdditionalArguments>
```

Comment:
now that procdump is available on CI we do not want to do this all the time as it really slows the build down. We only want to collect a dump on a crash or hang not normal execution.